lib/ui/hooks.dart

```diff
@@ -83,6 +83,11 @@ void _beginFrame(int microseconds) {
     window.onBeginFrame(new Duration(microseconds: microseconds));
 }
 
+void _drawFrame() {
```

Comment:
Named this _drawFrame for consistency with _beginFrame above.  Can change both if that's preferable.